src/display/editor/tools.js

```diff
@@ -555,6 +561,8 @@ class AnnotationEditorUIManager {
 
   #lastActiveElement = null;
 
+  #lastSelectedEditor = null;
+
```

Comment:
*One more question:* Given that manually tracking related state can make it easy for a bug to sneak in, with future code changes, could we perhaps just replace this field with a getter instead?

```javascript
get #lastSelectedEditor() {
  let editor = null;
  for (editor of this.#selectedEditors) {
    // Iterate to get the last element.
  }
  return editor;
}
```